index.html

```diff
@@ -3007,11 +3007,13 @@ <h3>Expiration</h3>
       </section>
 
       <section>
-        <h3>Revocation</h3>
+        <h3>Status Revocation</h3>
```

Comment:
I wouldn't rename the section, as it will probably confuse readers. What exactly is "status revocation"? I know what revocation is... but status revocation sounds like I'm revoking the status of the credential. Suggest moving it back to just "Revocation".

Reply:
... since we're invoking the `credentialStatus` property, I thought "Status Revocation" would be ok.  It's not a strongly held opinion... :)

Reply:
But the value of the credentialStatus property describes the status of the credential, i.e. whether or not it had been revoked. 
My understanding is that it is the credential that may be revoked, not the status of the credential.

Reply:
@brentzundel we're getting a little meta here....  The fact is the `credentialStatus` property is where we tell Verifiers to look for this information -- there isn't a `revocation` service...  As an issuer, if I issue and invalidate a credential, I would indicate that in the credentialStatus service in our specification (assuming I'm not doing something cool like you do w/ the distributed ledger, where I can just "disconnect" it :) )

Reply:
I don't think we're getting meta. Both @brentzundel and I are reading the title rename in the same way and we find it confusing. For example, why didn't you rename it to "Status Suspension" or "Status Retraction". Why did you pick "Revocation" as the title instead of those other two things, both of which are perfectly reasonable things to associate with the `credentialStatus` property.

Does that help explain why @brentzundel and I find the proposed renaming confusing?

Reply:
I'm going to merge this, and undo the title change... in an attempt to move things forward. Feel free to submit another PR if you feel the title change is necessary.